tests/_test_mscolab/test_models.py

```diff
@@ -25,9 +25,25 @@
     limitations under the License.
 """
 import pytest
+import datetime
 
 from mslib.mscolab.server import register_user
-from mslib.mscolab.models import User
+from mslib.mscolab.models import AwareDateTime, User, Permission, Operation, Message, Change
+
+
+class Test_AwareDateTime:
+    @pytest.fixture(autouse=True)
+    def setup(self):
+        self.aware_datetime = datetime.datetime.now(tz=datetime.timezone.utc)
+
+    def test_aware_datetime_conversion(self):
+        aware_datetime_type = AwareDateTime()
+
```

Comment:
check the value is None case too

Reply:
This is a good start and a definite improvement of the tests. To achieve a 100% coverage (a typical goal), one needs to write tests that trigger all LoC of the relevant functions. Here, the next step would be to supply None as datetime and expect None back to test the if-branch in the code.

More important, the functionality of the code should be tested. Here, the relevant functionality is a conversion from (potentially) local time to UTC, either with/without actual conversion.

Here, one could add tests with a time in, e.g., CET, generated by using  ```import pytz; pytz.timezone("CET")```